### ISSUE TITLE
[drone_teleop] Position control in new GUI doesn't work

### ISSUE THREAD

**Issue by diegomrt** [JdeRobot/drones#40]
It seems that the message never reaches the `/mavros/setpoint_raw/local` topic

**Comment by pariaspe** [comment]
To try the position control within the new GUI, `my_solution.py` file at each exercise must be modified. Modifications would be something like:

- At `main()` subscribe to `gui/pose`:
```
	rospy.Subscriber('gui/pose', Pose, gui_pose_cb)
```
- Also add `gui_pose_cb` method:
```
def gui_pose_cb(msg):
	global drone
	drone.set_cmd_pos(x=msg.position.x, y=msg.position.y, z=msg.position.z)
```

**Comment by pariaspe** [comment]
Fixed in JdeRobot/RoboticsAcademy#551
